compiler_gym/envs/llvm/service/LlvmSession.cc

```diff
@@ -130,7 +130,9 @@ Status LlvmSession::applyAction(const Action& action, bool& endOfEpisode,
     case LlvmActionSpace::PASSES_ALL:
       LlvmAction actionEnum;
       if (action.choice_size() != 1) {
-        return Status(StatusCode::INVALID_ARGUMENT, "Missing choice");
+        return Status(
+            StatusCode::INVALID_ARGUMENT,
+            fmt::format("Invalid choice count. Expected 1, received {}", action.choice_size()));
```

Comment:
This is just me being curious and not a review comment.
Why use `fmt::format` instead of an `f` string?

Reply:
C++ has no f strings :(

Reply:
Ahem, umm, oh, yeah. 
I can't think why you'd imagine this wasn't a thorough code review.